_modules/variables.md

```diff
@@ -9,4 +9,66 @@ order: 4
 
 ## What are Variables?
 
-##
\ No newline at end of file
+Variables are a way for us to store information in our program that we can change later. There are three parts to a variable: a **type**, a **name**, and a **value**. For example, I could store a variable in my program called "myFavoriteNumber" like this:
+
+```c
+int myFavoriteNumber = 8;
+```
+
+After I have _declared_ `myFavoriteNumber`, I can use it anywhere in my program. And if my favorite number changes later, I can change my variable to reflect that:
+
+```c
+myFavoriteNumber = 21;
+```
+
+Notice that I only have to tell the _compiler_ what type of variable myFavoriteNumber is when I first declare it. If I change the value later, I don't have to include the type; the _compiler_ will remember it automatically. `myFavoriteNumber` is an `int` type variable, which means that it can only be a whole number. If my favorite number was 3.14, this type wouldn't work! Some other common variable types include `double` (for decimal numbers), `bool` (for `true` or `false` values), and `char` (for single characters like the letter `'a'`).
+
+## Examples
+
+Let's add a few variables to our code. In your `variables_activity.cpp` file, in the `variables_exploration` function definition, add the following lines of code:
+
+```c
+int currentLed = 1;
+int currentNote = NOTE_C4;
+int duration = 1000;
+```
+
+We've now made three variables. Hopefully it's obvious what they represent based on their names. Notice that we can set our variables equal to other variables! `currentNote` has the same value as `NOTE_C4`. If we add `speaker_play_note(currentNote, duration)` to our code, it's the same as adding `speaker_play_note(NOTE_C4, 1000)`. Middle C will be played for one second.
+
+Let's use our variables to do something interesting. First, let's light up our current LED. Add the following to your code, and upload it to your board:
+
+```c
+leds_set_color(currentLed, 255, 0, 0);
+```
+
+It's nice that we don't have to hard-code the number 1 into our code to tell the _compiler_ that we want to light it up. It makes our code a little easier to read. We can even use it to light up other leds. Add the following to your code and reupload it to your board:
+
+```c
+currentLed++;
```

Comment:
Let's just use `currentLed = currentLed + 1` instead of `++`

Reply:
Ok. I figured it would make it easier to understand in the loops module if they'd already seen it, but I can take it out here and add the explanation there instead.

Reply:
Oh good call.  Sure let's leave it how you have it.  Can you add a statement explaining that both do the same thing?

Reply:
Will do.

Reply:
Done